astro.config.mjs

```diff
@@ -93,11 +93,11 @@ export default defineConfig({
               collapsed: true,
               autogenerate: { directory: "tutorials/json-editor" },
             },
-            {
-              label: "Async Counter App",
-              collapsed: true,
-              autogenerate: { directory: "tutorials/counter-async-app" },
-            },
+            // {
```

Comment:
Add a comment that docs why

Reply:
Resolved in https://github.com/ratatui-org/ratatui-website/pull/598/commits/5fe57244c71e3182919a9d095a70cec39f6bfb65